src/fields/configure/get-component-props.js

```diff
@@ -13,7 +13,7 @@
 /* eslint-disable max-len */
 import without from 'lodash/without';
 import { serializer } from '../components/lib/RichText';
-import getMuiProps from './get-mui-props';
+import getProps from './get-props';
```

Comment:
This is a breaking change @skipyT

Reply:
hi, I considered that it is ok, because the tests are passing and the getMuiProps is used only by the library. why do you consider this a breaking change?

Reply:
yes this one actually ties in with props coming from Material UI as defined here
- https://material-ui.com/api/text-field/#props

Which gives you full potential to pass in props from material ui.

Reply:
yes, indeed. I modified the get-mui-props, now has an extra parameter to mention the key, it is still working for mui:
`...getProps(uiSchema),
      ...getProps(uiSchema, 'custom')`      
and the function:
`export default (props, propsKey = 'mui') =>`
I think it is still supports the old cases.

Reply:
ok perfect this looks good - we can call it just `ui` then i think internal components can use this as well although i feel if we can centralize this with `ui:options` then thats the best solution.

Reply:
There is a open discussion here on how to have it done for reference 
- https://github.com/json-schema-org/json-schema-vocabularies/issues/2